virtcontainers/pkg/oci/utils.go

```diff
@@ -605,6 +605,11 @@ func ContainerConfig(ocispec CompatOCISpec, bundlePath, cid, console string, det
 			resources.VCPUs = uint32(utils.ConstraintsToVCPUs(*ocispec.Linux.Resources.CPU.Quota, *ocispec.Linux.Resources.CPU.Period))
 		}
 	}
+	if ocispec.Linux.Resources.Memory != nil {
+		if ocispec.Linux.Resources.Memory.Limit != nil {
+			resources.Mem = uint32(*ocispec.Linux.Resources.Memory.Limit)
```

Comment:
This is casting an int64 into an int32:

- https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md

Maybe `ContainerResources.Mem` should be changed to `int64` too?